libs/network/include/network/muddle/rpc/client.hpp

```diff
@@ -91,8 +90,10 @@ class Client : public service::ServiceClientInterface
 
   SharedHandler handler_;
 
+  std::mutex call_mutex_;  // priority 1
```

Comment:
What do the priorities mean in practice?

Reply:
The order in which the mutexs are locked in order not to deadlock